### PR TITLE
BTM-324: UK dates

### PR DESCRIPTION
**Pull request by gdsgj**
Puts dates in UK time zone instead of UTC, to match transaction data

### CODE REVIEW

**Review by Si3rr4wow — COMMENTED** [review]
Could we add some integration level testing to this too? Just a couple of events with relevant timestamps, could probably just tweak the existing test data

File: integration_tests/tests/athena-query-tests.ts [hunk 1]
```diff
@@ -16,6 +18,28 @@ describe("\nGenerate valid event and execute athena query\n", () => {
...
+describe("\nGenerate valid event one hour before January UTC and execute athena query\n", () => {
+  test("should not contain eventId in the generated query results for January", async () => {
+    const { eventId } = await invokeFilterLambdaAndVerifyEventInS3Bucket(
+      validEventPayloadOneHourBeforeJanuaryUtc
+    );
+    const queryString = `SELECT * FROM "btm_transactions_standardised" where month='01'`;
+    const queryResult = await queryAthena(queryString);
+    expect(JSON.stringify(queryResult)).not.toContain(eventId);
+  });
+});
```
Comment (markschnitziusgovuk) [inline integration_tests/tests/athena-query-tests.ts:41]: Considering also adding an event that's just outside the target month, and testing that it doesn't get returned.

Comment (markschnitziusgovuk) [reply integration_tests/tests/athena-query-tests.ts:41]: Sorry, that comment was meant for the preceding test.  But both tests maybe could maybe stand to have an 'inside' and 'outside' case.

Comment (markschnitziusgovuk) [reply integration_tests/tests/athena-query-tests.ts:41]: Resolved in dev standup.

**Pull request merged**